src/components/decorations/HR.tsx

```diff
@@ -0,0 +1,5 @@
+import SVG from "react-inlinesvg";
+
+export default function HR(): JSX.Element {
+  return <img src="/border.svg" alt="" />;
```

Comment:
You are not using the `<SVG>` component imported above ?

Reply:
Nope, my bad